sdf.md

```diff
@@ -642,18 +642,22 @@ All names in SDF are case-sensitive.
 
 # SDF structure
 
-SDF definitions are contained in SDF files.  One or more SDF files can
-work together to provide the definitions and declarations that are the
-payload of the SDF format.
+SDF definitions are contained in SDF documents, together with data
+about the SDF document itself (information block).
+Definitions and declarations from additional SDF documents can be
+referenced; together with the definitions and declarations in the
+referencing SDF document they build the model expressed by the
+referencing SDF document.
```

Comment:
As there are a number of instances of "SDF model" left in the draft, should it maybe be introduced as its own term in the Terminology section and also be mentioned here?

Reply:
I created a new issue
https://github.com/ietf-wg-asdf/SDF/issues/128

The "model" is mentioned in line 649, do you want to change this to "SDF model"?